packages/sanity/src/structure/panes/document/comments/CommentsWrapper.tsx

```diff
@@ -37,10 +38,11 @@ function CommentsProviderWrapper(props: CommentsWrapperProps) {
   const {children, documentId, documentType} = props
 
   const {enabled} = useCommentsEnabled()
-  const {connectionState, onPathOpen, inspector, openInspector} = useDocumentPane()
+  const {connectionState, onPathOpen, inspector, openInspector, version} = useDocumentPane()
   const router = useRouter()
   const {params, setParams, createPathWithParams, ...paneRouter} = usePaneRouter()
   const perspective = paneRouter.perspective ?? router.stickyParams.perspective
+  const docId = version ? getVersionId(documentId, version) : documentId
```

Comment:
Tiny nitpick: it seems potentially confusing to have both a `documentId` and a `docId` in scope with different meanings. Have you considered renaming `docId` to something like `resolvedDocumentId`, or passing this value directly below on line 87 to avoid binding it to a new name altogether?

Reply:
Renamed in an upcoming commit :)